reporter/src/main/kotlin/reporters/NoticeReporter.kt

```diff
@@ -43,11 +44,17 @@ class NoticeReporter : Reporter() {
         val footers: List<String>
     )
 
-    class PostProcessor(ortResult: OrtResult, noticeReport: NoticeReport, copyrightGarbage: CopyrightGarbage) :
+    class PostProcessor(
+        ortResult: OrtResult,
+        noticeReport: NoticeReport,
+        copyrightGarbage: CopyrightGarbage,
+        licenseConfiguration: LicenseConfiguration
+    ) :
         ScriptRunner() {
```

Comment:
The supertype should be on the same line as the closing paranthesis.

Reply:
Done.